fe/src/main/java/org/apache/doris/rewrite/FEFunctions.java

```diff
@@ -56,8 +49,13 @@
      */
     @FEFunction(name = "timediff", argTypes = { "DATETIME", "DATETIME" }, returnType = "TIME")
     public static FloatLiteral timeDiff(LiteralExpr first, LiteralExpr second) throws AnalysisException {
-        long timediff = (getTime(first) - getTime(second)) / 1000;
-        return new FloatLiteral((double)timediff, Type.TIME);
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
why "yyyy-MM-dd" other than "yyyy-MM-dd hh:mm:ss"

I think we can write a function getTime() for DateLiteral.